base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
Huh? I'm saying the part that you're adding here is unclear and would be better if phrased differently. "where `x` is the ..." doesn't make sense in this context.